### DISCUSSION TITLE
Error: Failed to complete swap

### DISCUSSION THREAD

**Discussion by 360Lookin** [comit-network/xmr-btc-swap#707] [General]
I attempted a BTC to XMR swap but received the following error:

Error: Failed to complete swap

Caused by:
    Seller blockchain network BlockchainNetwork { bitcoin: Testnet, monero: Stagenet } setup did not match your blockchain network setup BlockchainNetwork { bitcoin: Bitcoin, monero: Mainnet }

Will the BTC used in this swap return to the bitcoin-change-address in this case?

**Comment by fbion** [comment]
because this is Testnet🥶

**Reply by 360Lookin** [reply]
I realised that once this error occurred. Is there a protocol for the BTC to release to the bitcoin-change-address or is the BTC no longer retrievable in this scenario?

**Reply by thomaseizinger** [reply]
The BTC sit in the internal wallet. Once https://github.com/comit-network/xmr-btc-swap/issues/694 is resolved, you will be able to withdraw them.
